docs/getting-started/quickstart.md

```diff
@@ -19,7 +21,245 @@ Let's create your first DBOS app! You develop this app in a folder on your compu
   }}
 />
 
-In this guide, we start by deploying a sample "Hello" app to the cloud. After that, we show you how to run it locally.
+In this guide, you'll start by deploying a sample "Hello" app to the cloud. Then, you'll run it locally.
+
+
+<LargeTabs groupId="language">
+<LargeTabItem value="python" label="Python">
+
+
+## Deploy your First App to the Cloud
+
+### 1. Initialize your application
+
+In a clean directory, set up and activate a Python virtual environment:
+
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
```

Comment:
Have you tested this on Windows?

Reply:
Yeah, the whole DBOS Python needs some Windows testing, we'll do it next week.